docs/openapi.json

```diff
@@ -797,6 +797,20 @@
             "name": "block_size",
             "required": false,
             "type": "string"
+          },
+          {
+            "description": "Results page number (pagination).",
+            "in": "query",
+            "name": "page",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "description": "Number of results per page (pagination).",
+            "in": "query",
+            "name": "size",
+            "required": false,
+            "type": "string"
```

Comment:
Totally minor comment, not need to change. Shouldn't this and the previous parameter be integers? UI doesn't care about this, and REANA-Client doesn't use this (the difference would be to have a 1 as string or as int in the REANA-Client Python code).

Reply:
IIRC I left it as string because reana-client was sending it as string.. I can have another look

Reply:
Fixed and r-commons updated.